### ISSUE TITLE
Bootstrap 4 update

### ISSUE THREAD

**Issue by okendoken** [flatlogic/awesome-bootstrap-checkbox#63]
There are should be a separate branch of awesome-bootstrap-checkbox based on Bootstrap 4 version.

**Comment by kswope** [comment]
:+1: 

And how about a warning that bootstrap 4 isn't currently supported, took me a while to figure that out :(

**Comment by gaea44** [comment]
Add theses lines before importing  awesome-bootstrap-checkbox (scss version) :

```css
// preserve compatibility between awesome-bootstrap-checkbox and bootstrap 4
$input-border: $input-border-color;

@mixin scale($ratio...) {
  -webkit-transform: scale($ratio);
  -ms-transform: scale($ratio); // IE9 only
  -o-transform: scale($ratio);
  transform: scale($ratio);
}

@mixin transition-transform($transition...) {
  -webkit-transition: -webkit-transform $transition;
  -moz-transition: -moz-transform $transition;
  -o-transition: -o-transform $transition;
  transition: transform $transition;
}
```

**Comment by DimaSavchenko** [comment]
Hi, Guys ! We released new version v1.0.0-alpha.1 with support Bootstrap4.
Thanks !
